### PR TITLE
Bump to juno baseApp & update to the latest version

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: com.github.philip_scott.spice-up.json [hunk 1]
```diff
@@ -1,20 +1,14 @@
 {
     "id": "com.github.philip_scott.spice-up",
-    "base": "io.elementary.Loki.BaseApp",
-	"base-version": "stable",
+    "base": "io.elementary.BaseApp",
+	"base-version": "juno",
```
Comment (nedrichards) [inline com.github.philip_scott.spice-up.json]: indenting is inconsistent here.

**Comment by nedrichards** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
Fails to build. The app-id needs to be patched in the appdata file too.

**Comment by bilelmoussaoui** [comment]
@nedrichards Fixed that. Can you please trigger a new build? Thanks

**Comment by Philip-Scott** [comment]
Thank you all!

File: appdata-app-id.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
+--- a/data/com.github.philip-scott.spice-up.appdata.xml	2018-10-04 02:48:01.000000000 +0200
++++ b/data/com.github.philip-scott.spice-up.appdata.xml	2018-10-17 13:46:05.940049544 +0200
+@@ -1,7 +1,7 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <!-- Copyright (c) 2016 Felipe Escoto (https://github.com/Philip-Scott/Spice-up) -->
+ <component type="desktop">
+-  <id>com.github.philip-scott.spice-up</id>
++  <id>com.github.philip_scott.spice-up</id>
```
Comment (TingPing) [inline appdata-app-id.patch]: Add `<provides>` for old ID>

**Comment by nedrichards** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
@nedrichards I have fixed the build and updated the application to the latest release.

File: appdata-app-id.patch (hunk 2 above)
Comment (bilelmoussaoui) [reply appdata-app-id.patch]: Fixed

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
Depends on https://github.com/flathub/io.elementary.BaseApp/pull/9

**Comment by bilelmoussaoui** [comment]
Updated the baseApp.
bot, build

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by bilelmoussaoui** [comment]
@TingPing thanks for merging! I have sent a PR to update Notes-Up too https://github.com/flathub/com.github.philip_scott.notes-up/pull/4. Can you merge it too, please?

**Pull request merged**